lib/4_filter_numbers.php

```diff
@@ -14,7 +14,7 @@ function filterNumbers($arr) {
 		}
 	}
 	$arr = array_diff($arr, $big);
-	return array_multisort($arr);
+	return sort($arr);
```

Comment:
Hey! So I dug into it a little bit more and it turns out that `sort()` [returns](http://php.net/manual/en/function.sort.php) `true` if it "runs" successfully.

So, in your tests, I _think_ you're technically evaluating if `true == array(num, num, num....)`, which it does because the array is populated (!) but which doesn't test the functionality of the method.

Two possible fixes:

1. In your method, run `sort($arr);` on its own line. This line manipulates `$arr`. Then your final return can be `return $arr;` Make sure in your tests to compare against an array of the less-than-ten values ordered low to high.

2. If you're not that tied to sorting the array, simply `return $arr;` and in your tests compare against an array of the less-than-ten values in the order that they appeared in the original input array.